gtsam_unstable/geometry/Similarity3.h

```diff
@@ -182,10 +207,25 @@ class Similarity3: public LieGroup<Similarity3, 7> {
   /// @name Helper functions
   /// @{
 
-  /// Calculate expmap and logmap coefficients.
 private:
+  /// Calculate expmap and logmap coefficients.
```

Comment:
Only the ones important enough to be unit-tested should be exposed here. Other ones (maybe all!) should not be in header file, but just declared as static global functions in .cpp. The naming convention for those is camelCase. Finally, google style guide recommends to put all of those in an anonymous 
```
namespace {
}
``1st